axelor-project/src/main/resources/views/SprintPeriod.xml

```diff
@@ -25,4 +25,27 @@
 
   </form>
 
+  <form model="com.axelor.utils.db.Wizard" title="Sprint period generator"
+    name="sprint-period-generator-wizard-form" width="large">
+
+    <panel name="mainPanel">
+      <field name="company" title="Company" type="many-to-one"
+        target="com.axelor.apps.base.db.Company" required="true" x-dirty="false"/>
+      <spacer name="dateSpacer"/>
+      <field name="fromDate" title="From" type="date" required="true" x-dirty="false"/>
+      <field name="toDate" title="To" type="date" required="true" x-dirty="false"/>
```

Comment:
Need to add an action-validate as action-employee-validate-dates at the dates onChange and before the button onClick please